### PR TITLE
Add powershell APIs for Partition merge and Throughput redistribution

### PR DESCRIPTION
**Pull request by amar-sagare**
## Checklist

- [x ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x ] The title of the PR is clear and informative
- [x ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [x ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: NuGet.Config [hunk 1]
```diff
@@ -6,8 +6,10 @@
     <add key="local-feed" value="tools/LocalFeed" />
     <add key="azure-powershell" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-powershell/nuget/v3/index.json" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
+    <add key="PrivateNuget" value="E:\github\main\azure-sdk-for-net\artifacts\packages\Debug\mgmt" />
```
Comment (wyunchi-ms) [inline NuGet.Config:9]: Please remove the local repo.

File: src/CosmosDB/CosmosDB/help/Invoke-AzCosmosDBSqlContainerMerge.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> {{ Add example code here }}
+```
+
+{{ Add example description here }}
+
```
Comment (wyunchi-ms) [inline src/CosmosDB/CosmosDB/help/Invoke-AzCosmosDBSqlContainerMerge.md:45]: Please fill the placeholder here.

File: src/Security/Security.Test/ScenarioTests/SqlInformationProtectionPolicyTests.cs [hunk 3]
```diff
@@ -12,86 +12,79 @@
 // limitations under the License.
 // ----------------------------------------------------------------------------------
 
-using Microsoft.Azure.Commands.ScenarioTest;
-using Microsoft.Azure.ServiceManagement.Common.Models;
 using Microsoft.WindowsAzure.Commands.ScenarioTest;
 using Xunit;
```
Comment (wyunchi-ms) [inline src/Security/Security.Test/ScenarioTests/SqlInformationProtectionPolicyTests.cs:17]: Please remove the changes of other modules

File: src/CosmosDB/CosmosDB/SQL/InvokeAzCosmosDBSqlContainerMerge.cs [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            }
+
+            PhysicalPartitionStorageInfoCollection physicalPartitionStorageInfoCollection = null;
+            MergeParameters mergeParameters = new MergeParameters(isDryRun: false);
+
+            if (ShouldProcess(Name, "Merging partitions.", String.Empty, out ShouldProcessReason shouldProcessReason))
+            {
+                if (this.Force.IsPresent || ShouldContinue($"This command will merge the partitions of collection {Name} , do you want to continue?", String.Empty))
+                {
+                    physicalPartitionStorageInfoCollection =
```
Comment (ramarag) [inline src/CosmosDB/CosmosDB/SQL/InvokeAzCosmosDBSqlContainerMerge.cs:93]: can we add a message here like -" this operation might take a long time (potentially hours) depending on the data movement involved "

**Review by wyunchi-ms — DISMISSED** [review]

**Review by wyunchi-ms — APPROVED** [review]

**Pull request merged**